packages/create-app/templates/default-app/app-config.yaml.hbs

```diff
@@ -93,18 +93,10 @@ catalog:
         - allow: [Group, User]
 
     # Backstage example templates
-    - type: url
-      target: https://github.com/backstage/backstage/blob/master/plugins/scaffolder-backend/sample-templates/react-ssr-template/template.yaml
-      rules:
-        - allow: [Template]
     - type: url
       target: https://github.com/backstage/backstage/blob/master/plugins/scaffolder-backend/sample-templates/springboot-grpc-template/template.yaml
       rules:
         - allow: [Template]
-    - type: url
```

Comment:
Does this mean that we should have a note in the changeset, saying that we deprecated the use of this template and suggest removing it from one's config? Perhaps with some instructions on what's required to reinstate it or something similar to it